### PR TITLE
Feature/data explorer table

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR creates the Data explorer provider to import the data (now only of historical-emissions) and displays it on a table:

- Create Provider filtered by section
- Create selector for blacklist and parsing of data
- Display table
- Specify some first columns to keep years at the end
- Add links to associated modules and download

image

### CODE REVIEW

**Comment by simaob** [comment]
It works for me! =)

**Comment by Bluesmile82** [comment]
Added link to module and download

File: app/javascript/app/routes/app-routes/data-explorer-routes/data-explorer-routes.js [hunk 1]
```diff
@@ -19,25 +19,25 @@ export default [
...
-    label: 'NDC-SDG Linkages',
-    path: '/data-explorer/ndc-sdg-linkages',
-    component: () =>
-      createElement(DataExplorerContent, {
-        section: 'ndc-sdg-linkages'
-      }),
-    anchor: true,
-    exact: true
   }
+  // {
```
Comment (weberjavi) [inline app/javascript/app/routes/app-routes/data-explorer-routes/data-explorer-routes.js]: Can we delete those commented lines?

Comment (Bluesmile82) [reply app/javascript/app/routes/app-routes/data-explorer-routes/data-explorer-routes.js]: I have changed them on the next PR. Still, those will be the final routes when all the data is ready

**Pull request merged**